scripts/install/opt/seagate/cortx/motr/libexec/motr_mini_prov_logrotate.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+LOGROTATE_CONF_FILE=/etc/cortx/motr/mini_prov_logrotate.conf
+if [[ -f "$LOGROTATE_CONF_FILE" ]]; then
+    /usr/sbin/logrotate $LOGROTATE_CONF_FILE
+    EXITVALUE=$?
+fi
+
+if [ $EXITVALUE != 0 ]; then
```

Comment:
if EXITVALUE  is not defined then script will fail

Reply:
I have checked that $EXITVALUE will not be undefined even though it is never been assigned. It will have default value 0 assigned to it. Even though I have changed the code such that we will only check  $EXITVALUE where it is actually assigned with the return value of bash command.
Done 7230b13